hicommonwealth/commonwealth
# Added default page when there are no stake transactions

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/7320

## Description of Changes
- Added default page when there are no stake transactions on the `/myCommunityStake` page

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` in `.env`
- Visit `/myCommunityStake` page, and verify that you see the new default page when there are no transactions ([figma](https://www.figma.com/file/2IDrY6BAwADptR4AWB0Wjh/Portfolio-%26-Transactions?type=design&node-id=182-1685&mode=design&t=mgfIOuZfoAtU0LbE-0))
- Verify that you can see history table when there are any transactions

## Deployment Plan
N/A

## Other Considerations
N/A
